cmd/gokey/main.go

```diff
@@ -54,7 +56,16 @@ func genSeed(w io.Writer) {
 }
 
 func genPass(seed []byte, w io.Writer) {
-	password, err := gokey.GetPass(pass, realm, seed, &gokey.PasswordSpec{10, 3, 3, 1, 1, ""})
+	upperLower := math.Ceil(float64(passwordLength) * 3.0 / 10.0)
```

Comment:
I suggest not to hardcode this logic - what if there would be a weird website which will require "all uppercase"? It is not supported now, but I think all of them should be cmd line parameters.

Reply:
Excellent point - I'll work on that

Reply:
I wasn't sure how to do this with only the standard library since these cmd line parameters wouldn't always be used.